war/src/main/js/upgradeWizard.js

```diff
@@ -1,12 +1,11 @@
-onSetupWizardInitialized(function(wizard) {
-	var $ = wizard.$; // jQuery
+function onSetupWizardInitialized(wizard) {
```

Comment:
:bug: No, please revert this file and just add `/* global onSetupWizardInitialized */` to the first line